src/SmartTransactionsController.ts

```diff
@@ -423,29 +512,31 @@ export default class SmartTransactionsController extends BaseController<
   // ! Ask backend API to accept list of uuids as params
   async fetchSmartTransactionsStatus(
     uuids: string[],
+    { networkClientId }: { networkClientId?: NetworkClientId } = {},
   ): Promise<SmartTransaction[]> {
```

Comment:
This return type seems to be inaccurate.

```suggestion
  ): Promise<Record<string, SmartTransactionsStatus>> {
```

Reply:
done here: https://github.com/MetaMask/smart-transactions-controller/pull/237/commits/93a05f14cbf91a344332dcf07d39f6924d5eacd3